Tabloid/Repositories/CommentRepository.cs

```diff
@@ -0,0 +1,113 @@
+
+        public List<Comment> GetAllByPostId(int postId)
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"
+                 SELECT Id, PostId, UserProfileId, Subject, Content, CreateDateTime FROM Comment
+                 WHERE PostId = @postId
+                 ";
+
+                    DbUtils.AddParameter(cmd, "@postId", postId);
+
+                    using (SqlDataReader reader = cmd.ExecuteReader())
+                    {
+
+                        var comments = new List<Comment>();
+                        while (reader.Read())
+                        {
+                            comments.Add(new Comment()
+                            {
+                                Id = DbUtils.GetInt(reader, "Id"),
+                                PostId = DbUtils.GetInt(reader, "PostId"),
+                                UserProfileId = DbUtils.GetInt(reader, "UserProfileId"),
+                                Subject = DbUtils.GetString(reader, "Subject"),
+                                Content = DbUtils.GetString(reader, "Content"),
+                                CreateDateTime = DbUtils.GetDateTime(reader, "CreateDateTime")
+                            });
+                        }
+
+                        return comments;
+                    }
+                }
+            }
+        }
+
+        public void Add(Comment comment)
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"INSERT INTO Comment (PostId, UserProfileId, Subject, Content, CreateDateTime)
+                                        OUTPUT INSERTED.ID
+                                        VALUES (@PostId, @UserProfileId, @Subject, @Content, @CreateDateTime)";
+                    DbUtils.AddParameter(cmd, "@PostId", comment.PostId);
+                    DbUtils.AddParameter(cmd, "@UserProfileId", comment.UserProfileId);
+                    DbUtils.AddParameter(cmd, "@Subject", comment.Subject);
+                    DbUtils.AddParameter(cmd, "@Content", comment.Content);
+                    DbUtils.AddParameter(cmd, "@CreateDateTime", comment.CreateDateTime);
+
+                    comment.Id = (int)cmd.ExecuteScalar();
+                }
+            }
+        }
+
+        public void Delete(int id)
+        {
+            using (var conn = Connection)
+            {
+                conn.Open();
+                using (var cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = "DELETE FROM Comment WHERE Id = @Id";
+                    DbUtils.AddParameter(cmd, "@id", id);
+                    cmd.ExecuteNonQuery();
+                }
+            }
+        }
+        public void Update(Comment comment)
```

Comment:
Looks like this is unrelated to this issue ticket. Please remove this code until the appropriate issue is being addressed.